src/posix/platform/radio_spinel.hpp

```diff
@@ -439,6 +439,18 @@ class RadioSpinel : public HdlcInterface::Callbacks
     otError PlatDiagProcess(const char *aString, char *aOutput, size_t aOutputMaxLen);
 #endif
 
+    /**
+     * This method returns the radio channel mask.
+     *
+     * @param[in]   aPreferred  TRUE to get preferred channel mask, FALSE to get supported channel mask.
+     *
+     * @returns The radio channel mask according to @aPreferred:
+     *   The radio supported channel mask that the device is allowed to be on.
+     *   The radio preferred channel mask that the device prefers to form on.
+     *
+     */
+    uint32_t GetRadioChannelMask(bool aPreferred);
```

Comment:
Declare as `const`?

Reply:
We can't declare as const here, since this function will call into RadioSpinel::Get() which will change the member variables.